torch/_dynamo/variables/base.py

```diff
@@ -30,7 +30,7 @@ class ParentsTracker:
 
     def __init__(self):
         # logically this is a set, but we use a dict to ensure deterministic ordering
-        self.parents: Dict[ParentsTracker, bool] = dict()
+        self.parents: Dict[ParentsTracker, bool] = {}
```

Comment:
I have a ruff rule for this is we want. to enable it codebase wide.

Reply:
I'd be fine with this.